src/components/ModalConfirm.vue

```diff
@@ -145,6 +147,12 @@ watch(
             <BaseIcon name="info" size="24" class="text-skin-text" />
           </BaseLink>
         </div>
+        <div class="flex">
+          <span class="mr-1 flex-auto text-skin-text" v-text="$t('reason')" />
+        </div>
+        <div class="flex">
+          <TextareaAutosize v-model="reason" :max-length="140" />
```

Comment:
We should have `class="s-input !rounded-3xl"` on TextareaAutosize. I see you found many different stylings for Textareas, these are mostly left overs that need to be refactored